File: Basic/Conditions/219158M.c
#include <stdio.h>
#define intll long long

void solution() {

	char c;
	scanf("%c", &c);

	if (c >= '0' && c <= '9') {
		printf("IS DIGIT\n");
	} else {
		printf("ALPHA\n");
		if (c >= 'a' && c <= 'z') {
			printf("IS SMALL\n");
		} else if (c >= 'A' && c <= 'Z') {
			printf("IS CAPITAL\n");
		}
	}
}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Conditions/219158M.go
package main

import (
	"fmt"
	"unicode"
)

func solution() {

	var c rune
	fmt.Scanf("%c", &c)

	if unicode.IsDigit(c) {
		fmt.Printf("IS DIGIT\n")
	} else {
		fmt.Printf("ALPHA\n")
		if unicode.IsLower(c) {
			fmt.Printf("IS SMALL\n")
		} else if unicode.IsUpper(c) {
			fmt.Printf("IS CAPITAL\n")
		}
	}
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
